server/test/integration/routes/events.generateSeating.spec.js

```diff
@@ -0,0 +1,65 @@
+const request = require('supertest');
+const proxy = require('proxyquire');
+const dbConfig = require('../../config/db.js');
+const seeder = require('../../database/seed');
+const utils = require('../utils');
+
+dbConfig['@global'] = true;
+dbConfig['@noCallThru'] = true;
+describe.skip('integration: events - generateSeating', () => {
```

Comment:
Since I've requested changes elsewhere, might as well ask for this:

add a comment explaining why it's skipped